src/main/cpp/Robot.cpp

```diff
@@ -77,16 +77,19 @@ Robot::Robot() : frc::TimesliceRobot{2_ms, Constants::kControllerPeriod} {
                                  [=] { AutoTargetZoneShootThree(); });
     m_autonChooser.AddAutonomous("Target Zone Shoot Six",
                                  [=] { AutoTargetZoneShootSix(); });
-    m_autonChooser.AddAutonomous("Right Side Drive Forward",
-                                 [=] { AutoRightSideDriveForward(); });
     m_autonChooser.AddAutonomous("Right Side Intake",
                                  [=] { AutoRightSideIntake(); });
     m_autonChooser.AddAutonomous("Right Side Shoot Three",
                                  [=] { AutoRightSideShootThree(); });
-    m_autonChooser.AddAutonomous("Right Side Shoot Six",
-                                 [=] { AutoRightSideShootSix(); });
     m_autonChooser.AddAutonomous("Right Side Shoot Eight",
                                  [=] { AutoRightSideShootEight(); });
+
+    m_autonChooser.AddAutonomous("Drive Forward",
```

Comment:
I meant that the AddAutonomous() calls that weren't used for Tidal Tumble would get moved from here to AutonomousTest.cpp in the test fixture's constructor (you'll need to add a constructor).